src/enzo/macros_and_parameters.h

```diff
@@ -325,6 +342,7 @@ typedef long long int   HDF5_hid_t;
 #define HDF5_FILE_PINT HDF5_FILE_I4
 #define PISYM "d"
 #define ENPY_PINT NPY_INT
+#define EYT_PINT YT_INT
```

Comment:
For consistency with the rest of this file, both of these should probably have their respective `ifdefs` unless it breaks something about compilation to do so.

Reply:
I'm reasonably certain I was just imitating what I'd done before for `ENPY_PINT` but I think it's worth investigating.

Reply:
`ifdef` is added here.